docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,147 @@
+---
+meta:
+  title: "Webhooks"
+  parentTitle: "Guides"
+  description: "Learn Liveblocks Webhooks concepts and implementation"
+---
+
+## Webhooks
+
+In this guide, we will explain how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data real time. Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
+
+### What is a webhook?
+The main drivers of webhooks implementation are efficiency and observability. Webhooks streamline collecting data between two connected services. 
+
+Like an API, webhooks are a mechanism for a client to recieve data. The main difference between webhooks and APIs is that a webhook functions as an inverted API. In essence, the architecture of a webhook is composed of the following parts: 
```

Comment:
```suggestion
Like an API, webhooks are a mechanism for a client to receive data. The main difference between webhooks and APIs is that a webhook functions as an inverted API, you give us an endpoint that we can send events to. In essence, the architecture of a webhook is composed of the following parts: 
```

inverted API is unclear, what do you think about adding a sentence to explain what it means? / adding an example.

Reply:
good on clarification.  How generic do you want example to be? I initially had one in and removed since I felt the concept was straightforward- but an easy one I encounter during the holidays is online shopping + out of stock items---> subscribe to in stock notifications.   vs the polling i would be doing if that option isn't available. Did you have something particular in mind?